### ISSUE TITLE
Coroutine object returned by jupyter_kernel_mgmt

### ISSUE THREAD

**Issue by bnavigator** [jupyter/jupyter_kernel_test#52]
Is this project still active? We have a bunch of packages in openSUSE which still have it in their dependency tree. As a policy, python packages should activate unit tests wherever possible.

To avoid flit as build frontend, I tried to update to current master so I can use pip for PEP517 style building.

Unfortunately jkt doesn't seem to be compatible with recent Python asyncio behavior and latest jupyter_kernel_mgmt:

```
[  113s] + /usr/bin/python3 test_ipykernel.py
[  114s] /home/abuild/rpmbuild/BUILD/jupyter_kernel_test-f33ca3bdea9b0b46526bb7385752760f0ca6d736/jupyter_kernel_test/__init__.py:36: RuntimeWarning: coroutine 'KernelFinder.launch' was never awaited
[  114s]   conn_info, km = cls.launch_kernel()
[  114s] RuntimeWarning: Enable tracemalloc to get the object allocation traceback
[  114s] E
[  114s] ======================================================================
[  114s] ERROR: setUpClass (__main__.IPyKernelTests)
[  114s] ----------------------------------------------------------------------
[  114s] Traceback (most recent call last):
[  114s]   File "/home/abuild/rpmbuild/BUILD/jupyter_kernel_test-f33ca3bdea9b0b46526bb7385752760f0ca6d736/jupyter_kernel_test/__init__.py", line 36, in setUpClass
[  114s]     conn_info, km = cls.launch_kernel()
[  114s] TypeError: cannot unpack non-iterable coroutine object
```

**Comment by maveme** [comment]
@bnavigator were you able to fix this issue? I'm having the same trouble. However, if I install it using pip, it works.

**Comment by bnavigator** [comment]
Hi, AFAICS, (not fully sure what I did more than a month ago), I abandoned the idea to update to current master.

Here is the current package with version 0.3: It builds with pip as frontend after calling flit.tomlify:
https://build.opensuse.org/package/show/devel:languages:python:jupyter/python-jupyter_kernel_test

By "it works", do you mean the test suite passes, or that you can use it without having it tested? Do you use the released version when using pip or a version from the git repo?

**Comment by blink1073** [comment]
Fixed by #57
